build.sbt

```diff
@@ -102,7 +107,7 @@ lazy val finagle = project
   .settings(moduleName := "catbird-finagle")
   .settings(allSettings)
   .settings(
-    libraryDependencies += "com.twitter" %% "finagle-core" % finagleVersion,
+    libraryDependencies += "com.twitter" %% "finagle-core" % (tlBaseVersion.value + ".0"),
```

Comment:
Ideally we'd get Scala Steward PRs that would bump `tlBaseVersion` for us, but I'm not sure that will work as long as [`tlBaseVersion must be of form x.y`](https://github.com/typelevel/sbt-typelevel/blob/66b313c7bdd7c6a8f35b5d6c41c9c4f9a19a773b/versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala#L68). Open to suggestions—the goal is to tie the `tlBaseVersion` value to the Finagle and Twitter Util version.

Reply:
Maybe like this?

```scala
val finagleVersion = "whatever"
ThisBuild / tlBaseVersion = finagleVersion.processIt()
```